### PR TITLE
feat(icons): add expand and collapse icons

### PR DESCRIPTION
**Pull request by shridhar52war**

### CODE REVIEW

File: src/icons/Contract/Contract.native.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (kamleshchandnani) [inline src/icons/Contract/Contract.native.js:1]: this should be collapse right? Expand, collapse

Comment (shridhar52war) [reply src/icons/Contract/Contract.native.js:1]: I just went with Expand's antonym Contract 😛 Was confused in choosing between shrink, condense, contract. Collapse, not sure if it appropriate.

File: src/icons/index.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ const icons = {
   chevronLeft: require('./ChevronLeft').default,
   chevronRight: require('./ChevronRight').default,
   chevronUp: require('./ChevronUp').default,
+  contract: require('./Contract').default,
```
Comment (chaitanyadeorukhkar) [inline src/icons/index.js:13]: Are you making it alphabetical? :P

Comment (dhruvdutt) [reply src/icons/index.js:13]: Yes, please.

File: src/icons/Contract/Contract.native.js (hunk 1 above)
Comment (dhruvdutt) [reply src/icons/Contract/Contract.native.js:1]: Expand <> Collapse sounds better.

**Pull request merged**